rubocop/rubocop.yml

```diff
@@ -62,3 +62,12 @@ Style/TrailingCommaInArrayLiteral:
 
 Style/TrailingCommaInHashLiteral:
   EnforcedStyleForMultiline: comma
+
```

Comment:
Thoughts on incorporating @zfletch's [suggestion](https://github.com/TheGnarCo/gnar-style/issues/48#issuecomment-595265444)?

> Unrelated to the cops themselves but related to the configuration: I think it would be a good idea in rubocop.yml to separate these cops from the other cops and to leave a comment about them.
> 
> When Rubocop enables these cops by default in the next major release, they can then be removed. Otherwise it will become difficult to tell pending cops from cops intentionally configured differently from the default.

Reply:
I still think this would be a good idea unless/until RuboCop has functionality to remove redundant cops. One of the benefits of YAML compared with JSON or another format is that it allows comments.

Reply:
@danfrenette have you had an opportunity to look into this?

Reply:
Not recently no, I've set a reminder for next week so if I don't get to it by EOD I'll be able to do it then. Thanks for the heads up!